File: app/modules/ytdlp/android/src/main/java/expo/modules/ytdlp/platforms/TikTok.kt
package expo.modules.ytdlp.platforms

class TikTok(url: String) : BasePlatform(url) {
    companion object {
        val HOSTS = setOf("tiktok.com", "www.tiktok.com", "vm.tiktok.com", "vt.tiktok.com")
    }
}

File: app/server/platforms/TikTok.ts
import { BasePlatform } from "./BasePlatform.js";

export class TikTok extends BasePlatform {
  readonly id = "tiktok";

  static readonly HOSTS = ["tiktok.com", "www.tiktok.com", "vm.tiktok.com", "vt.tiktok.com"];
}
